linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/oval/shared.xml

```diff
@@ -4,6 +4,8 @@
 {{% set smartcard_packages = ['pam_pkcs11', 'mozilla-nss', 'mozilla-nss-tools', 'pcsc-ccid', 'pcsc-lite', 'pcsc-tools', 'opensc'] %}}
 {{% elif product in ["rhel7", "ol7"] %}}
 {{% set smartcard_packages = ['pam_pkcs11'] %}}
+{{% elif product in ["ubuntu2004"] %}}
```

Comment:
no need of this, see rule.yml
we use the templated oval, not that one.

Reply:
Hi, this is correct but OCIL description, and fixtext use this variable, should this still be removed?
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/rule.yml

Reply:
Realizing that this is templated... I've removed this branch condition for ubuntu. Thanks for the feedback.